CRM/Contribute/Form/Contribution/Confirm.php

```diff
@@ -934,7 +934,7 @@ public static function processFormContribution(
         $pledgeParams['create_date'] = $pledgeParams['start_date'] = $pledgeParams['scheduled_date'] = date("Ymd");
```

Comment:
@mlutfy @eileenmcnaughton I don't want to cause more work for JMA ;) but shouldn't date formats in reports be drawn from civicrm/admin/setting/date?action=reset=1 ?